rules/windows/defense_evasion_execution_suspicious_explorer_winword.toml

```diff
@@ -0,0 +1,45 @@
+[metadata]
+creation_date = "2020/09/03"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/09/03"
+
+[rule]
+author = ["Elastic"]
+description = """
+An instance of Explorer or Winword was started after being renamed or from unusual standard path. This is uncommon
+behavior and may indicate an attempt to run unnoticed or undetected via side loading a malicious DLL within the memory
+space of one of those processes. Verify any related child process, network connections and image loads events.
+"""
+from = "now-9m"
+index = ["winlogbeat-*", "logs-endpoint.events.*"]
+language = "kuery"
+license = "Elastic License"
+name = "Potential DLL SideLoading via Trusted Microsoft Programs"
+risk_score = 73
+rule_id = "1160dcdb-0a0a-4a79-91d8-9b84616edebd"
+severity = "high"
+tags = ["Elastic", "Windows"]
+type = "query"
+
+query = '''
+event.category:process and not event.type:end and
+ process.pe.original_file_name:(WinWord.exe or EXPLORER.EXE) and
+ not (process.name:(winword.exe or WINWORD.EXE or explorer.exe) or
+             process.executable:("C:\Windows\explorer.exe" or C\:\\Program*Files*\\Microsoft*Office\\root\\Office*\\WINWORD.EXE))
```

Comment:
Since both `\` and `\\` work, what is the preference for consistency @brokensound77?

Reply:
IIRC when we use wildcard * in KQL need to escape all special chars including \ : etc.

Reply:
yea, preferably quoted with single slash, but as Samir stated, wildcards would require no quotes and so escaping everything

Reply:
these _should_ (please double check me) translate to very performant queries.

`C\:\\Program?Files*\\Microsoft?Office\\root\\Office*\\WINWORD.EXE` and
`C\:\\Program?Files?\(x86\)\\Microsoft?Office\\root\\Office*\\WINWORD.EXE`

once we have EQL, and don't have the weird escaping issues here, it'll be a little prettier to read

Reply:
nice sugg works 
![image](https://user-images.githubusercontent.com/64742097/92303119-3bfd7d00-ef72-11ea-9b84-5136f2dfbbe9.png)